charts/pega/values-large.yaml

```diff
@@ -183,6 +183,10 @@ global:
           # For GKE Pre-shared Certificate, use ingress.gcp.kubernetes.io/pre-shared-cert: <pre-shared-certificate-name>
           # For GKE to use static IP for load balancer, use kubernetes.io/ingress.global-static-ip-name: <global-static-ip-name>
           ssl_annotation:
+          # for enabling TLS in Openshift Route as Reencrypt, add certificate, key and CaCertificate PEM or crt format.
```

Comment:
for enabling TLS in Openshift Route as Reencrypt, add certificate, key and CaCertificate PEM or crt format.

can we update this wording to be less jargony? maybe not all clients would understand as written. As a start, I suggest>
For Openshift, enable TLS in Route do this [whatever Reencrypt specifically means in this context], and add your certificate, the corresponding key using the appropriate .pem or .crt format.

Reply:
also wondering why this is only in large-values.yaml and not in the basic values.yaml file.